content/tracing/services/_index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Services list
+kind: Documentation
+---
+
+## Overview
+
+After having [instrumented your application](/tracing/languages), your reporting services are listed on [the APM services page](https://app.datadoghq.com/apm/services).  
```

Comment:
+After you have [instrumented your application](/tracing/languages), your reporting services will appear on [the APM services page](https://app.datadoghq.com/apm/services).  Select a service to view detailed performance insights ([read our dedicated service documentation to learn more](tracing/services/service)).

Reply:
👍